packages/framework/src/services/smb/smb-broker.client.test.ts

```diff
@@ -0,0 +1,102 @@
+import { BrokerClientFactory } from '@pequehq/smb-client';
+import * as sinon from 'sinon';
+import { suite } from 'uvu';
+import * as assert from 'uvu/assert';
+
+import { BrokerAddressInvalidError } from '../../models';
+import { SmbBrokerClient, SmbPublishPayload, SmbSubscribePayload } from './smb-broker.client';
+
+const test = suite('SMB Broker Client');
+
+test.before.each(async (context) => {
```

Comment:
We can remove `async` here and the `after.each` below since we reassign sandbox already